test/regress/regress0/model-core.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --produce-model-cores
+; COMMAND-LINE: --produce-models --model-cores=simple
```

Comment:
Is there a test case where the model core is predictable and robust, in which we can use ;EXPECT in order to verify that the model core is what we expected it to be?

Reply:
Is it possible to implement something similar to `--check-models` for model cores (maybe out-of-scope for this PR)?